packages/paypal-js/src/v6/index.ts

```diff
@@ -1,13 +1,64 @@
-import { loadCustomScript as originalLoadCustomScript } from "../load-script";
+import { insertScriptElement } from "../utils";
 
 const version = "__VERSION__";
 
-// V6-specific loadCustomScript without PromisePonyfill
-function loadCustomScript(options: {
-    url: string;
-    attributes?: Record<string, string>;
-}): Promise<void> {
-    return originalLoadCustomScript(options); // Uses default Promise
+type LoadScriptOptions = {
+    environment?: "production" | "sandbox";
+    debug?: boolean;
+};
+
+function loadCoreSdkScript(options: LoadScriptOptions = {}) {
+    validateArguments(options);
+    const { environment, debug } = options;
+
+    // resolve with null when running in Node or Deno
+    if (typeof document === "undefined") return Promise.resolve(null);
+
+    const baseURL =
+        environment === "sandbox"
+            ? "https://www.sandbox.paypal.com"
+            : "https://www.paypal.com";
```

Comment:
Would it be safer to default to `sandbox`?
```suggestion
    const baseURL =
        environment === "production"
            ? "https://www.paypal.com"
            : "https://www.sandbox.paypal.com";
```

Reply:
Nice suggestion @bywood. I agree, let's default to sandbox instead. I updated the PR with this change.